boa/src/syntax/parser/statement/block.rs

```diff
@@ -0,0 +1,201 @@
+//! <https://tc39.es/ecma262/#sec-block>
```

Comment:
Better documentation.
```suggestion
//! Block statement parsing.
//!
//! More information:
//!  - [MDN documentation][mdn]
//!  - [ECMAScript specification][spec]
//!
//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block
//! [spec]: https://tc39.es/ecma262/#sec-block
```